test/utils/test_normalization.py

```diff
@@ -0,0 +1,183 @@
+"""Test normalization."""
+
+import numpy as np
+import scipy.sparse.linalg as spl
+from scipy.sparse import csr_matrix, diags
+
+from toponetx.utils.normalization import (
+    _compute_B1_normalized_matrix,
+    _compute_B1T_normalized_matrix,
+    _compute_B2_normalized_matrix,
+    _compute_B2T_normalized_matrix,
+    _compute_D1,
+    _compute_D2,
+    _compute_D3,
+    _compute_D5,
+    compute_bunch_normalized_matrices,
+    compute_kipf_adjacency_normalized_matrix,
+    compute_laplacian_normalized_matrix,
+    compute_x_laplacian_normalized_matrix,
+    compute_xu_asymmetric_normalized_matrix,
+)
+
+
+def test_compute_laplacian_normalized_matrix():
+    """Test normalize laplacian."""
+    adjacency_matrix = np.array(
+        [
+            [0, 1, 0, 0, 0, 1],
+            [1, 0, 1, 0, 0, 0],
+            [0, 1, 0, 1, 0, 0],
+            [0, 0, 1, 0, 1, 0],
+            [0, 0, 0, 1, 0, 1],
+            [1, 0, 0, 0, 1, 0],
+        ]
+    )
+
+    # Calculate the degree matrix
+    degree_matrix = np.diag(adjacency_matrix.sum(axis=1))
+
+    # Calculate the Laplacian matrix
+    L = np.array(degree_matrix - adjacency_matrix)
+
+    L = csr_matrix(L).asfptype()
+    normalized_L = compute_laplacian_normalized_matrix(L)
+    expected_result = csr_matrix(
```

Comment:
No need to construct a `csr_matrix` here when you convert it to dense arrays below again. Just use `np.array`.